content/docs/09.administrator-guide/backup-and-restore.md

```diff
@@ -5,13 +5,71 @@ icon: /docs/icons/admin.svg
 
 Backup Kestra.
 
-Kestra does not provide specific tools for backup and restore. However, Kestra uses a database and internal storage that can be backed up and restored.
+Kestra only provides a backup tool for Kestra __metadata__: all things that are not execution data.
+However, Kestra uses a database and internal storage that can be backed up and restored if a metadata backup is not enough.
 
-## Backup & Restore with the JDBC Backend
+## Metadata only Backup & Restore in the Enterprise Edition
 
-With the JDBC backend, Kestra can be backed up and restored using the database's native backup tools. 
+Since version 0.19, Kestra [Enterprise Edition](/enterprise) provides __metadata__ backup & restore.
+This can be used to backup Kestra metadata in a Kestra instance, and restore them in another instance that can be in a different version or using a different backend.
 
-### Backup & Restore for PostgreSQL
+It is strongly advised to backup and restore metadata when Kestra is stopped or the backup may not be in a consistency state.
+
+Currently, metadata backup will backup everything that is not related to Kestra execution data, which includes at the time of writing this documentation: custom blueprints, flows, namespaces, roles, secrets (for JDBC and Elasticsearch secrets), security integrations, settings, templates, tenants, triggers and users.
+
+### Metadata backup
+
+To backup Kestra instance metadata you can use the following command:
+
+```shell
+kestra backups create FULL
+```
+
+`FULL` indicate that we want to backup the full instance, if you want to backup only a single tenant (in case multi-tenancy is enabled), you can use `TENANT` instead. In this case, users and tenants will not be included in the backup (only the selected tenant will be included).
```

Comment:
```suggestion
`FULL` indicates that we want to back up the full instance. If you want to back up only a single tenant (in case multi-tenancy is enabled), you can use `TENANT` instead. In this case, users and tenants will not be included in the backup (only the selected tenant will be included).
```